### ISSUE TITLE
Missing alt / title parameters from FIP image tag

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#9]
Missing alt / title parameters from FIP image tag
==> Add description: "Government of Canada / Gouvenrement du Canada".
